contracts/ERC721A.sol

```diff
@@ -316,20 +316,26 @@ contract ERC721A is Context, ERC165, IERC721, IERC721Metadata, IERC721Enumerable
             addressData.balance + uint128(quantity),
             addressData.numberMinted + uint128(quantity)
         );
-        _ownerships[startTokenId] = TokenOwnership(to, uint64(block.timestamp));
 
-        uint256 updatedIndex = startTokenId;
+        uint256 batches = quantity / maxBatchSize;
```

Comment:
I think if we do go with this change, we should rename `maxBatchSize` to `batchSize` since it's not really a constraint, and more of a parameter used to optimize the mint process